Manufacturer_smartron_rimo02a

```diff
@@ -0,0 +1,18 @@
+<manufacturer id="Manufacturer">
+        <DeviceCode>
+            <maintainer>DevendarReddyD</maintainer>
+            <devicename>SmartronSRT</devicename>
+            <filename>crDroidAndroid-9.0-20190324-rimo02a-v5.2</filename>
+            <buildtype>Monthly</buildtype>
+            <download>http://bit.ly/2JPWU1R</download>
+            <changelog>ChangelogLink</changelog>
+            <gapps>https://opengapps.org/</gapps>
+            <forum>https://forum.xda-developers.com/member.php?u=9815186</forum>
+            <firmware>FirmwareLink</firmware>
```

Comment:
If you don't have firmware, modem, bootloader, recovery links, then just delete it
Those tags are optional